src/lib.rs

```diff
@@ -86,22 +87,28 @@ impl FdPassingExt for RawFd {
             iov_len: mem::size_of_val(&dummy),
         };
 
-        let msg: msghdr = libc::msghdr {
-            msg_name: std::ptr::null_mut(),
-            msg_namelen: 0,
-            msg_iov: &mut iov,
-            msg_iovlen: 1,
-            msg_control: unsafe { u.buf.as_mut_ptr() as *mut c_void },
-            msg_controllen: msg_len,
-            msg_flags: 0,
-        };
+        let mut msg: MaybeUninit<msghdr> = MaybeUninit::uninit();
```

Comment:
```suggestion
        let mut msg: MaybeUninit<msghdr> = MaybeUninit::zeroed();
```

i think this may be a bit safer and more portable.